dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TestUtils.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.logging.internal_fat;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ *
+ */
+public class TestUtils {
+
+    private static Class<?> c = TestUtils.class;
+
+    public static void runApp(LibertyServer server, String type) {
+        String url = "http://" + server.getHostname() + ":" + server.getHttpDefaultPort() + "/MpTelemetryLogApp";
+        if (type.equals("access")) {
+            url = url + "/AccessURL";
+        } else if (type.equals("ffdc1")) {
+            url = url + "?isFFDC=true";
+        } else if (type.equals("ffdc2")) {
+            url = url + "?secondFFDC=true";
+        } else if (type.equals("ffdc3")) {
+            url = url + "?thirdFFDC=true";
+        } else if (type.equals("trace")) {
+            url = url + "/TraceURL";
+        } else if (type.equals("logServlet")) {
+            url = url + "/LogURL";
+        } else if (type.equals("extension")) {
+            url = url + "/ExtURL";
+        } else if (type.equals("exception")) {
+            url = url + "/ExceptionURL";
```

Comment:
Good to add a test, how an exception is bridged over. Need to investigate if an FFDC is created in this cases as well?